asset-transfer-basic/chaincode-external/README.md

```diff
@@ -0,0 +1,55 @@
+# Asset-Transfer-Basic as an external service
+
+See the "Chaincode as an external service" documentation for running chaincode as an external service.
+This includes details of the external builder and launcher scripts which peers in your Fabric network will require.
+
+The Asset-Transfer-Basic chaincode requires two environment variables to run, `CHAINCODE_SERVER_ADDRESS` and `CHAINCODE_ID`, which are described in the `chaincode.env.example` file. Copy this file to `chaincode.env` before continuing.
+
+**Note:** each organization in a Asset-Transfer-Basic network will need to follow the instructions below to host their own instance of the Asset-Transfer-Basic external service.
+
+## Packaging and installing
+
+Make sure the value of `CHAINCODE_SERVER_ADDRESS` in `chaincode.env` is correct for the Asset-Transfer-Basic external service you will be running.
+
+The peer needs a `connection.json` configuration file so that it can connect to the external Asset-Transfer-Basic service.
+Use the `CHAINCODE_SERVER_ADDRESS` value in `chaincode.env` to create the `connection.json` file with the following command (requires [jq](https://stedolan.github.io/jq/)):
+
+```
+env $(cat chaincode.env | grep -v "#" | xargs) jq -n '{"address":env.CHAINCODE_SERVER_ADDRESS,"dial_timeout": "10s","tls_required": false}' > connection.json
+```
+
+Add this file to a `code.tar.gz` archive ready for adding to a Asset-Transfer-Basic external service package:
+
+```
+tar cfz code.tar.gz connection.json
+```
+
+Package the Asset-Transfer-Basic external service using the supplied `metadata.json` file:
+
+```
+tar cfz asset-transfer-basic-pkg.tgz metadata.json code.tar.gz
+```
+
+Install the `asset-transfer-basic-pkg.tgz` chaincode as usual, for example:
+
+```
+peer lifecycle chaincode install ./asset-transfer-basic-pkg.tgz
+```
+
```

Comment:
As communicated on the status call this step actually requires an external builder that is currently missing. The phrasing here is rather misleading because it is not quite "as usual". :-)